.github/workflows/lint.yml

```diff
@@ -0,0 +1,14 @@
+# .github/workflows/main.yml
```

Comment:
Why this?

Reply:
It was there with default snippet. Should I remove it?

Reply:
Or change it to something actually describing this file, comment should be meaningful.